scripts/rook.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash -E
 
-ROOK_VERSION=${ROOK_VERSION:-"v1.2.7"}
+ROOK_VERSION=${ROOK_VERSION:-"v1.4.9"}
 ROOK_DEPLOY_TIMEOUT=${ROOK_DEPLOY_TIMEOUT:-300}
 ROOK_URL="https://raw.githubusercontent.com/rook/rook/${ROOK_VERSION}/cluster/examples/kubernetes/ceph"
 ROOK_BLOCK_POOL_NAME=${ROOK_BLOCK_POOL_NAME:-"newrbdpool"}
-ROOK_CEPH_CLUSTER_VERSION="v14.2.10"
+ROOK_CEPH_CLUSTER_VERSION=${ROOK_CEPH_CLUSTER_VERSION:-${BASE_IMAGE}}
```

Comment:
`BASE_IMAGE` contains `ceph/ceph:v15` we need to extract the version `v15.x.x`  from the base image and use it here.

Reply:
How will we determine x?

Reply:
Sometimes the version might not be in `vx.y.z` format  Just extract the version after `:` and use it?

Reply:
ok